src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (num1) => {
+    return (num2) => {
```

Comment:
You can set 0 as a default value so if no argument is passed, it would still work. But you code it sound.